pkg/pluginmanager_service/plugin_manager.go

```diff
@@ -156,12 +156,15 @@ func (m *PluginManager) Get(req *pb.GetRequest) (*pb.GetResponse, error) {
 
 func (m *PluginManager) RefreshConnections(*pb.RefreshConnectionsRequest) (*pb.RefreshConnectionsResponse, error) {
 	resp := &pb.RefreshConnectionsResponse{}
+	go m.doRefresh()
+	return resp, nil
+}
+
+func (m *PluginManager) doRefresh() {
 	refreshResult := connection.RefreshConnections(context.Background())
 	if refreshResult.Error != nil {
-		return nil, refreshResult.Error
+		log.Printf("[WARN] RefreshConnections failed with error: %s", refreshResult.Error.Error())
```

Comment:
find issue about sending errors/warnings back from plugin manager to CLI and reference it here